DesktopModules/Vanjaro/Core/Library/Managers/SettingManager.cs

```diff
@@ -444,6 +444,10 @@ private static void DoIUpgradeable(PortalInfo pinfo)
                     UpdatePortalSettings(SeoSettings, pinfo.PortalID, uInfo.UserID);
                     #endregion
                 }
+                else
+                {
+                    HostController.Instance.Update("DisableEditBar", "False");
+                }
```

Comment:
This needs to be updated to support upgrades. It should happen in IUpgradeable targeting a specific version.

Reply:
changes updated